stripe/_stripe_response.py

```diff
@@ -61,5 +61,5 @@ def __init__(
     def stream(self) -> AsyncIterable[bytes]:
         return self._stream
 
-    async def read(self) -> bytes:
+    async def read(self) -> bytes:  # noqa: ASY100
```

Comment:
nit: consider filing a ticket and/or adding a TODO (MAJOR) comment to fix this naming in the next major.

Reply:
sure! added a `TODO (MAJOR)`.